### ISSUE TITLE
MarkDownFile not work!

### ISSUE THREAD

**Issue by kehehee1** [atishay/vscode-allautocomplete#6]
I install this plugin. When I open a markdown file, it just can't show word suggestions. But when I open a text file, it works fine. Help!

After tried a few times, I found that if set setting like below, it worked finaly.
```
"[markdown]": {
    "editor.quickSuggestions": true
```

**Comment by atishay** [comment]
Thanks for the information. I will add this information to the Readme

**Comment by atishay** [comment]
Added this information to the readme

**Comment by W4RH4WK** [comment]
Hey there, did you change any of the default configurations lately? At the moment, the extension does not take the regular `editor.quickSuggestions` option into account for Markdown.

I typically have quick suggestions disabled as I bring them up only when I need them. But today I noticed that they are popping up in Markdown files due to this extension while not having quick suggestions enabled. I have only disabled them *toplevel* not specifically for Markdown.

If I also add the following entry to my config, quick suggestions are also disabled for markdown:

```json
    "[markdown]": {
        "editor.quickSuggestions": {
            "other": false,
            "comments": false,
            "strings": false
        }
    }
```
